bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TypedescValueImpl.java

```diff
@@ -66,6 +67,11 @@ public TypedescValueImpl(Type describingType, MapValue[] closures) {
         this.closures = closures;
     }
 
+    @Deprecated
+    public TypedescValueImpl(Type describingType, MapValue[] closures, MapValue annotations) {
```

Comment:
Do we know why the constructors in this class are deprecated? And if we really need to do so with the new one?

Reply:
Earlier we haven't had api classes so these were use as api. Since these classes are internal now, you do not need to deprecated. IMO we need to remove in others as well. Can u create an issue for that.